app/javascript/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Layout from './components/Layout';
+import Home from './routes/Home';
+import Vehicles from './routes/Vehicles';
+import Details from './routes/Details';
+import Reservations from './routes/Reservations';
+import NewReserve from './routes/NewReserve';
+import Login from './routes/Login';
+import Signup from './routes/Signup';
+import NewVehicle from './routes/NewVehicle';
+import DeleteVehicle from './routes/DeleteVehicle';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.



You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 5):

![image](https://github.com/Luiscarlosvd/futuristic-transportation/assets/57421823/2a59ec66-35a4-4c60-8ed6-4ed9428f6e29)

That way, your code will meet the latest standards. 💪